crit/Makefile

```diff
@@ -37,9 +38,23 @@ gen-proto:
 		$(proto_files)
 
 bin/crit: cmd/cli.go
-	$(GO) build -o $@ $^
+	$(GO) build ${GOFLAGS} -o $@ $^
+
+../test/loop/loop:
```

Comment:
This is something I saw already earlier. We now have two test cases. piggie and loop. Is there a big difference? Maybe something for a future PR. Use the same thing for everything.

Reply:
Sure. Let's fix this in another pull request when we merge @snprajwal's changes from https://github.com/checkpoint-restore/go-criu/pull/109.